apps/masterbots.ai/app/api/og/route.tsx

```
@@ -1,53 +1,52 @@
+				question={question}
+				answer={answer}
+				username={username}
+				user_avatar={user_avatar}
+				isLightTheme={isLightTheme}
+			/>,
+			{
+				width: 1200,
+				height: 627,
+			},
+		)
+	} catch (e: any) {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)